common/core/desktop/include/keyman/keyboardprocessor_vkeys.h

```diff
@@ -35,6 +35,7 @@ enum km_kbp_modifier_state {
     KM_KBP_MODIFIER_NOSCROLLOCK = 1 << 13,
     KM_KBP_MODIFIER_VIRTUALKEY  = 1 << 14,
   */
+  KM_KBP_MODIFIER_RELEASE     = 1 << 15,
```

Comment:
This is tricky. This value is used in other places for `VIRTUAL_CHAR_KEY`. This is also a 16 bit value so we can't really use up this value.

Structurally, I also think I'd prefer to keep this out of the modifier bitmap because it's not really a modifier. Instead, why don't we update the `ProcessEvent` signature to add a `isKeyDown` parameter (I'd prefer `isKeyDown` to `isKeyUp`: I think that `true` for the active state is probably slightly more intuitive than the other way around).

Reply:
done

Reply:
Need to add this to the [15.0 wiki](https://github.com/keymanapp/keyman/wiki/Keyman-15.0-New-Functionality) as well.